### PR TITLE
Failure to Invalidate Session on 2FA activation

### PR DESCRIPTION
**Pull request by RRudder**

### CODE REVIEW

File: submissions/description/broken_authentication_and_session_management/failure_to_invalidate_session/on_two_fa_activation_change/template.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+## Business Impact
+
+This vulnerability can lead to reputational damage and indirect financial loss to the company as customers may view the application as insecure.
+
+## Steps to Reproduce
+
+1. Sign into a user’s account (Browser A)
+1. Sign into the same user’s account, using a different browser (Browser B)
+1. Using Browser A, set up 2FA for the account
+1. Using Browser B, observe that the user session is still valid
```
Comment (evildaemond) [inline submissions/description/broken_authentication_and_session_management/failure_to_invalidate_session/on_two_fa_activation_change/template.md:18]: Part of the steps to reproduce would be to trigger an authenticated action, for example, change the profile name or details, which would usually require an authenticated session.

**Review by evildaemond — CHANGES_REQUESTED** [review]
1 Technical change then LGTM

File: submissions/description/broken_authentication_and_session_management/failure_to_invalidate_session/on_two_fa_activation_change/template.md (hunk 1 above)
Comment (RRudder) [reply submissions/description/broken_authentication_and_session_management/failure_to_invalidate_session/on_two_fa_activation_change/template.md]: Thanks @evildaemond GTG

**Review by evildaemond — APPROVED** [review]
LGTM

**Pull request merged**